sources/models/statuses.js

```diff
@@ -1,4 +1,11 @@
-export const statuses = [
-	{"id":1,"Name":"Busy","Icon":"cogs"},
-	{"id":2,"Name":"Open","Icon":"user"}
-];
\ No newline at end of file
+// export const statuses = [
+// 	{"id":1,"Name":"Busy","Icon":"cogs"},
+// 	{"id":2,"Name":"Open","Icon":"user"}
+// ];
```

Comment:
remove commented lines

Reply:
I know it, sorry. Tested a lot of different features and in the end forgot to remove them.